src/glados/tui.py

```
@@ -137,17 +112,118 @@ def _display_next_char(self) -> None:
+        """Clear the content area."""
+        content_area = self.query_one("#content_area", RichLog)
+        content_area.clear()
+
+
+class LLMTextInput(Container):
+    """Text input widget for manual LLM interaction."""
+
+    def compose(self) -> ComposeResult:
+        with Vertical():
+            yield Static(
+                "[dim]Type to GLaDOS... (Enter to send... ug doesnt work Ctrl+S to send, Shift+Enter for new line, Esc to hide)[/dim]", 
```

Ruff (0.11.9)

Line too long (136 > 120)

(E501)